src/Plugins/Rhetos.LightDMS/LightDMSRhetosServiceCollectionBuilderExtensions.cs

```diff
@@ -20,6 +20,7 @@
 using Microsoft.AspNetCore.Mvc.ApplicationParts;
 using Microsoft.AspNetCore.Mvc.Controllers;
 using Microsoft.AspNetCore.StaticFiles;
+using Rhetos;

```

Comment:
I think this is not neeed?

Reply:
We need to add this when build LightDMS with latest code of Rhetos.